### ISSUE TITLE
Could not load some plugins

### ISSUE THREAD

**Issue by cheuberg** [indico/indico-containers#13]
I cloned the repository, renamed `indico.env.sample` to `indico.env` and said `docker-compose up`.
Then I got the following error messages (and http://localhost:8080/ says "internal server error"):
```
indico-web_1       | ERROR:  relation "events.events" does not exist
indico-web_1       | LINE 1: SELECT COUNT(*) FROM events.events
indico-web_1       |                              ^
indico-web_1       | Preparing DB...
indico-web_1       | Using PostgreSQL container...
indico-web_1       | /opt/indico/run_indico.sh: line 17: [: ==: unary operator expected
indico-web_1       | CREATE EXTENSION
indico-web_1       | CREATE EXTENSION
indico-web_1       | /opt/indico/.venv/lib/python2.7/site-packages/indico/core/config.py:184: UserWarning: Ignoring unknown config key OS
indico-web_1       |   warnings.warn('Ignoring unknown config key {}'.format(key))
indico-web_1       | /opt/indico/.venv/lib/python2.7/site-packages/indico/core/config.py:184: UserWarning: Ignoring unknown config key AST
indico-web_1       |   warnings.warn('Ignoring unknown config key {}'.format(key))
indico-web_1       | 2018-10-04 11:35:09,056  ERROR    0000000000000000  indico.plugins            Plugin  does not exist
indico-web_1       | Traceback (most recent call last):
indico-web_1       |   File "/opt/indico/.venv/bin/indico", line 11, in <module>
indico-web_1       |     sys.exit(cli())
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/core.py", line 722, in __call__
indico-web_1       |     return self.main(*args, **kwargs)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/flask/cli.py", line 380, in main
indico-web_1       |     return AppGroup.main(self, *args, **kwargs)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/core.py", line 697, in main
indico-web_1       |     rv = self.invoke(ctx)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
indico-web_1       |     return _process_result(sub_ctx.command.invoke(sub_ctx))
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/indico/cli/util.py", line 119, in invoke
indico-web_1       |     return self._impl.invoke(ctx)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/core.py", line 1063, in invoke
indico-web_1       |     Command.invoke(self, ctx)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/core.py", line 895, in invoke
indico-web_1       |     return ctx.invoke(self.callback, **ctx.params)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/core.py", line 535, in invoke
indico-web_1       |     return callback(*args, **kwargs)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
indico-web_1       |     return f(get_current_context(), *args, **kwargs)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
indico-web_1       |     return f(get_current_context(), *args, **kwargs)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/flask/cli.py", line 256, in decorator
indico-web_1       |     with __ctx.ensure_object(ScriptInfo).load_app().app_context():
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/flask/cli.py", line 229, in load_app
indico-web_1       |     rv = self.create_app(self)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/indico/cli/util.py", line 37, in _create_app
indico-web_1       |     return make_app(set_path=True)
indico-web_1       |   File "/opt/indico/.venv/lib/python2.7/site-packages/indico/web/flask/app.py", line 371, in make_app
indico-web_1       |     raise Exception('Could not load some plugins: {}'.format(', '.join(plugin_engine.get_failed_plugins(app))))
indico-web_1       | Exception: Could not load some plugins: 
```

**Comment by ThiefMaster** [comment]
Do you have the latest version? 0386ed57735d2814450d998d32a521fd32e3a1dd looks like it fixes this problem.

**Comment by cheuberg** [comment]
Yes:
```
$ git log -1
commit 0386ed57735d2814450d998d32a521fd32e3a1dd (HEAD -> master, origin/master, origin/HEAD)
Author: Pedro Ferreira <pedro.ferreira@cern.ch>
Date:   Thu Sep 20 10:45:29 2018 +0200

    Fix splitting of plugin names
```

**Comment by pferreir** [comment]
Fixed by cce1a10563175c4c1bf2e046bc53ebc820161670, thanks for the report!
